Source/WebCore/platform/graphics/FontCascade.h

```diff
@@ -319,7 +319,7 @@ class FontCascade : public CanMakeWeakPtr<FontCascade> {
         return advancedTextRenderingMode();
     }
 
-    FontCascadeDescription m_fontDescription;
+    mutable FontCascadeDescription m_fontDescription;
```

Comment:
Unfortunate we need to make the `FontCascadeDescription` just for `font-size-adjust: from-font`, but maybe @anttijk or @litherum has better ideas.

Reply:
Would you say more? Do you mean we need to create and use a clone of `FontCascadeDescription` for `font-size-adjust: from-font` in `FontCascade`? Even in that case, we may need `mutable` to avoid the `const` method chain.

Reply:
I'll defer to Myles/Antti, also ok with addressing this in a follow up.

Reply:
O.K. Let's go with this change for now. I will follow up on other's comments if any.